controller.h

```diff
@@ -374,13 +383,16 @@ class Controller : public IController
                 feature->Socket()->Stop();
     }
 
-    void Start() override
+    void Start(bool respectWantsToRun) override
```

Comment:
Added now in case we want to allow the Controller to be stopped and started using the API at a later time. In that case we will not want to consider whatever an effects manager prefers concerning its running state.